src/main/resources/labels/commercial-refrigeration-direct-sales/commercial-refrigeration-direct-sales.svg

```diff
@@ -0,0 +1,238 @@
+  <text class="cls-13" transform="translate(15.1626 276.49518) scale(1 1.012)">G</text>
+  <polygon class="cls-14" points="79.492 19.856 79.492 25.274 92.213 25.274 92.213 30.817 79.492 30.817 79.492 37.038 92.758 37.038 92.758 42.581 73.673 42.581 73.673 14.186 92.758 14.186 92.758 19.856 79.492 19.856"/>
+  <polygon class="cls-14" points="120.34 14.187 120.34 42.582 113.392 42.582 101.715 23.708 101.715 42.582 95.895 42.582 95.895 14.187 102.843 14.187 114.52 33.315 114.52 14.187 120.34 14.187"/>
+  <polygon class="cls-14" points="129.464 19.856 129.464 25.274 142.185 25.274 142.185 30.817 129.464 30.817 129.464 37.038 142.73 37.038 142.73 42.581 123.645 42.581 123.645 14.186 142.73 14.186 142.73 19.856 129.464 19.856"/>
+  <path class="cls-14" d="M151.601,26.92444h6.571a10.21581,10.21581,0,0,0,2.051-.169,3.4633,3.4633,0,0,0,1.172-.425q.63-.507.63-2.58a8.20644,8.20644,0,0,0-.377-3.089c-.253-.565-1.074-.847-2.472-.847h-7.575Zm-5.858,15.657v-28.395h13.394a15.20913,15.20913,0,0,1,4.017.465,6.43188,6.43188,0,0,1,2.764,1.50305,6.23788,6.23788,0,0,1,1.612,2.751,14.51451,14.51451,0,0,1,.521,4.168,19.48678,19.48678,0,0,1-.167,2.58,14.6502,14.6502,0,0,1-.46,2.202,4.27181,4.27181,0,0,1-1.338,1.903,5.55869,5.55869,0,0,1,1.234,2.116,8.78911,8.78911,0,0,1,.396,2.709v7.998H161.73v-7.364a2.67461,2.67461,0,0,0-.751-2.2,3.8712,3.8712,0,0,0-2.387-.592h-6.991v10.156Z"/>
+  <path class="cls-14" d="M180.9435,26.9248h13.601c.14,1.07306.214,1.86805.229,2.389.016.523.02.982.02,1.376v2.328a17.16018,17.16018,0,0,1-.919,5.84,5.00286,5.00286,0,0,1-1.946,2.517,8.55178,8.55178,0,0,1-3.706,1.207q-1.25552.126-2.592.169c-.896.027-1.787.043-2.678.043q-6.69748,0-9.21-2.159-2.721-2.2425-2.721-8.845v-1.777q0-.88944.043-1.819v-3.343q0-5.502,2.218-8.126a7.08416,7.08416,0,0,1,3.2-2.03,15.21582,15.21582,0,0,1,4.796-.677h1.545c.502,0,1.02.013,1.549.041h2.931q4.0995.12753,5.754,2.222a9.4087,9.4087,0,0,1,1.654,5.947v2.495h-5.858a12.51642,12.51642,0,0,0-.464-4.444q-.21-.42142-1.755-.507-1.296-.08248-2.534-.105c-.822-.013-1.682-.021-2.573-.021q-3.22351,0-3.807.677-.58952.72007-.67,4.697c0,.281-.007.549-.023.803-.011.254-.019.53705-.019.846v1.61l.042,3.595c0,.734.028,1.427.082,2.074a10.54431,10.54431,0,0,0,.292,1.777v-.084a2.18254,2.18254,0,0,0,.67.826,2.81814,2.81814,0,0,0,1.339.486,12.57154,12.57154,0,0,0,1.716.126c.335.03.678.044,1.028.044h2.487a20.9813,20.9813,0,0,0,2.367-.107,2.07468,2.07468,0,0,0,1.234-.444c.389-.339.588-1.466.588-3.385v-.677h-7.91Z"/>
+  <polygon class="cls-14" points="202.728 14.059 209.186 14.059 206.622 22.221 213.649 21.069 203.583 43.059 206.622 27.118 199.879 27.407 202.728 14.059"/>
+  <line class="cls-5" x1="8.504" y1="295.05321" x2="263.547" y2="295.05321"/>
+  <line class="cls-5" x1="8.504" y1="351.57112" x2="263.547" y2="351.57112"/>
+  <text id="supplier" class="cls-15" transform="translate(8.50391 72.04694)">X</text>
+  <text id="model" class="cls-16" transform="translate(263.8 71.98981)" text-anchor="end">X</text>

```

Comment:
It looks like we've lost the multi line supplier name and model name in the new labels. Can they still take the same approximate number of characters? They do seem a bit wider.
If not it may be tricky to change the length validation on categories with both old multi-line labels and new single line labels active as they would currently share the same generic supplier and name fields.

Reply:
Yeah good point. We might be able to go up to two rows for some of them - there's nothing in the regulations that prevent that. Though for electronic displays the QR code is taller than the flag which means there's not room for the model number to go over two lines. Probably going to be easier to test and adjust once they're in the generator so I'll merge and look at these when we're doing the form changes